server/server.js

```diff
@@ -1,25 +1,25 @@
 const express = require('express');
-const bodyParser = require('body-parser');
+// const bodyParser = require('body-parser');
 const db = require('../db/mongodb');
 // const fs = require('fs');
 // const _ = require('underscore');
 
 const app = express();
 
-app.use(bodyParser.json());
+// app.use(bodyParser.json());
 
 app.use(express.static(`${__dirname}/../react/dist`));
 
 app.get('/restaurants/:id', (request, response) => {
   response.set({ 'Access-Control-Allow-Origin': '*' });
```

Comment:
Not sure if you will need to set the headers. I took mine out and it works fine for now.

Reply:
Yeah this was in the original code.  I think it might be necessary for the proxy (when AJAX requests come in from a different location)